manim/mobject/numbers.py

```diff
@@ -286,17 +286,14 @@ def get_value(self):
 
 
 class Variable(VMobject, metaclass=ConvertToOpenGL):
```

Comment:
Personally I think Variable is a bad class, but I've done my best to fix the confusions it was causing me.

I've removed references to "python variable". The class doesn't relate to python variables at all - the value is actually stored in a ValueTracker.

Reply:
I like your proposed changes a lot, thank you. I agree that `Variable` is an excellent candidate for being renamed.